substitutions/js/background.js

```diff
@@ -58,6 +58,37 @@ var default_replacements = [
   ['urged restraint by', 'drunkenly egged on'],
   ['horsepower', 'tons of horsemeat'],
 
+  // https://xkcd.com/1679/
+  ['gaffe', 'magic spell'],
+  ['gaffes', 'magic spells'],
+  ['ancient', 'haunted'],
+  ['star-studded', 'blood-soaked'],
+  ['remains to be seen', 'will never be known'],
+  ['silver bullet', 'way to kill werewolves'],
+  ['silver bullets', 'ways to kill werewolves'],
+  ['subway system', 'tunnels I found'],
+  ['subway systems', 'tunnels I found'],
+  ['surprising', 'surprising (but not to me)'],
+  ['war of words', 'interplanetary war'],
+  ['wars of words', 'interplanetary wars'],
+  ['tension', 'sexual tension'],
+  ['cautiously optimistic', 'delusional'],
+  ['doctor who', 'the big bang theory'],
+  ['win votes', 'find pok\u00E9mon'],
+  ['behind the headlines', 'beyond the grave'],
+  ['email', 'poem'],
+  ['emails', 'poems'],
+  ['facebook post', 'poem'],
+  ['facebook posts', 'poems'],
+  ['tweet', 'poem'],
+  ['tweets', 'poems'],
+  ['facebook ceo', 'this guy'],
+  ['latest', 'final'],
+  ['disrupt', 'destroy'],
+  ['meeting', 'm\u00E9nage \u00E0 trois'],
+  ['scientist', 'Channing Tatum and his friends'],
```

Comment:
This should be 'scientists', in the plural.

Per your PR comment, I'd say 'scientist' in the singular should map to `Channing Tatum's friend` to avoid likely collisions with `Channing Tatum` in the wild. 😃